docs/Getting Started/yo-generator.md

```diff
@@ -134,19 +134,19 @@ For Keystone to provide authentication and session management to your applicatio
 - If you want your application to support session management, set the `session` option to true. Loading sessions incurs a small overhead, so if your application doesn't need sessions you can turn this off.
```

Comment:
Are there actually use cases for no sessions .. perhaps this is referring to persistent sessions versus in-memory?